test/strategies.js

```diff
@@ -2056,6 +2056,156 @@ const createLiquityPaybackChickenOutStrategy = () => {
     return strategy.encodeForDsProxyCall();
 };
 
+const createMorphoAaveV2FLBoostStrategy = () => {
+    const strategy = new dfs.Strategy('MorphoAaveV2FLBoostStrategy');
+
+    strategy.addSubSlot('&ratioState', 'uint256');
+    strategy.addSubSlot('&targetRatio', 'uint256');
+
+    strategy.addTrigger(new dfs.triggers.MorphoAaveV2RatioTrigger(
+        '%nullAddr', '%0', '%0',
+    ));
+    strategy.addAction(new dfs.actions.flashloan.FLAction(
+        new dfs.actions.flashloan.EulerFlashLoanAction(
+            '%dAsset', '%flAmount',
+        ),
+    ));
+    strategy.addAction(new dfs.actions.basic.SellAction(
+        formatExchangeObj(
+            '%dAsset',
+            '%cAsset',
+            '%exchangeAmount',
+            '%exchangeWrapper',
+        ),
+        '&proxy',
+        '&proxy',
+    ));
+    strategy.addAction(new dfs.actions.basic.GasFeeAction(
+        '%gasCost', '%cAsset', '$2',
+    ));
+    strategy.addAction(new dfs.actions.morpho.MorphoAaveV2SupplyAction(
```

Comment:
`from` and `onBehalf` should not be left to be injected from the backend side, as for instance `onBehalf` might be the attackers address. Both can be hardcoded with `&proxy`